### DISCUSSION TITLE
Windows build

### DISCUSSION THREAD

**Discussion by sergeevabc** [davidsmorais/kuro#63] [Help] [answered]
Dear @davidsmorais, I'm Windows 7 x64 user coming from [AO 6.9](https://github.com/klaudiosinani/ao/releases/tag/v6.9.0). As you see, that repo has setup.exe, however yours repo has none. Perhaps you could be so kind to generate one for Kuro? Even .zip with binary contents would be fine.

**Comment by davidsmorais** [comment]
@sergeevabc thanks for the message. 
In all honesty, I did not account for old windows versions users when I decided not to support `.exe` after forking AO 😓  
The reason is that I really thought all Windows machines did support the official ToDo app and I don't see any advantages of using Kuro over the official app (unless you're a mega fan of the Dracula theme 🧛 )

I will try and get an `.exe` build ASAP so you can use Kuro.

**Comment by davidsmorais** [comment, marked as answer]
@sergeevabc you can find the `.exe` on the [latest release](https://github.com/davidsmorais/kuro/releases/tag/v8.1.6).
Will include a `.exe` build from now on :)

**Reply by sergeevabc** [reply]
@davidsmorais, it seems there is no ```.exe``` over there.

![Screenshot](https://user-images.githubusercontent.com/373718/199735473-5c410776-2115-45ef-9f2e-ab19207d1065.png)

**Reply by davidsmorais** [reply]
@sergeevabc sorry about that, seems there was an error while uploading the `.exe`.
The `.exe` build show now be listed on the Releases page. 

Here's [the link](https://github.com/davidsmorais/kuro/releases/download/v8.1.6/Kuro.Setup.8.1.6.exe) :wink:

**Reply by sergeevabc** [reply]
@davidsmorais, thank you, it works as intended.
Your Dracula theme variation complements those in Firefox and VSCodium perfectly.
I hope you've no plans to ruin its beauty with screaming political banners in the sidebar. :face_with_spiral_eyes:

**Reply by davidsmorais** [reply]
Thanks :) 
And don't worry, the content inside Kuro is handled by Microsoft , I just tweak the colors :)
